pkg/controller/volume/persistentvolume/scheduler_bind_cache_metrics.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolume
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+// VolumeSchedulerSubsystem - subsystem name used by scheduler
+const VolumeSchedulerSubsystem = "volumeScheduler"
+
+var (
+	VolumeBindingRequestSchedulerBinderCache = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: VolumeSchedulerSubsystem,
+			Name:      "volume_binder_cache_requests_total",
+			Help:      "Total number for request volume binding cache",
+		},
+		[]string{"operation"},
+	)
+	VolumeSchedulingStageLatency = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Subsystem: VolumeSchedulerSubsystem,
+			Name:      "volume_scheduling_duration_microseconds",
```

Comment:
According to Prometheus best practices, we should use "seconds" instead of "microseconds".